webapp/security/models.py

```diff
@@ -7,13 +8,40 @@
     JSON,
     String,
     Table,
+    Enum,
 )
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship
 
 
 Base = declarative_base()
 
+# Enums
+
+
+class CveStatus(enum.Enum):
```

Comment:
I though you were going to remove the enums?

Reply:
No, just temporarily removing it from the insertion, although I will try again when I have time, because the problem was with alembic.

Reply:
What insertion? If the problem is with alembic does that not mean we are going to run into it whenever we need to apply a migration? Will things not fail when sqlalchemy tries to do things with the enums and the enum type is not actually defined on the DB schema?

Reply:
I removed the enums. Basically the problem is that autogenerate does not create the types.